docs/content/api/geoh5py.groups.rst

```diff
@@ -2,42 +2,42 @@ geoh5py.groups
 ==============
 
 
-geoh5py.groups.base\_group
---------------------------
-
-.. automodule:: geoh5py.groups.base
-   :members:
-   :undoc-members:
-   :show-inheritance:
-
 geoh5py.groups.container\_group
 -------------------------------
 
-.. automodule:: geoh5py.groups.container
+.. automodule:: geoh5py.groups.container_group
```

Comment:
I'm not sure about this

Reply:
I vote that we just re-create the PR as is once you we figure out the pathspec issue.